src/v1/interfaces.v20230901.ts

```diff
@@ -1,33 +1,26 @@
-import * as v20220901 from './interfaces.v20220901';
-
-export type Corporation = v20220901.Corporation;
-export type Facet = v20220901.Facet;
-export type AddressSearcherQuery = v20220901.AddressSearcherQuery;
-export type AddressSearcherOptions = v20220901.AddressSearcherOptions;
-export type NTACorporateInfoProcess = v20220901.NTACorporateInfoProcess;
-export type NTACorporateInfoKind = v20220901.NTACorporateInfoKind;
-export type NTACorporateInfoCloseCause = v20220901.NTACorporateInfoCloseCause;
-export type NTACorporateInfo = v20220901.NTACorporateInfo;
-export type NTACorporateInfoResolverResponse =
-  v20220901.NTACorporateInfoResolverResponse;
-export type NTACorporateInfoSearchMode = v20220901.NTACorporateInfoSearchMode;
-export type NTACorporateInfoSearcherOptions =
-  v20220901.NTACorporateInfoSearcherOptions;
-export type NTACorporateInfoFacets = v20220901.NTACorporateInfoFacets;
-export type NTACorporateInfoSearcherResponse =
-  v20220901.NTACorporateInfoSearcherResponse;
-
 import * as v20221101 from './interfaces.v20221101';
```

Comment:
Consider adding a comment explaining why the types are being exported from `v20221101` instead of `v20220901`. This will help future developers understand the reasoning behind this change. [medium]